File: Bars.Gkh/libs/B4/model/Dict/UnitMeasure.js
Ext.define('B4.model.dict.UnitMeasure', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'UnitMeasure'
    },
    fields: [
        { name: 'Id', useNull: true },
        { name: 'Name' },
        { name: 'ShortName' },
        { name: 'Description' },
        { name: 'OkeiCode' }
    ]
});

File: Bars.Gkh/Entities/Dicts/UnitMeasure.cs
namespace Bars.Gkh.Entities
{
    /// <summary>
    /// Единица измерения
    /// </summary>
    public class UnitMeasure : BaseGkhEntity
    {
        /// <summary>
        /// Наименование
        /// </summary>
        public virtual string Name { get; set; }

        /// <summary>
        /// Короткое наименоваание
        /// </summary>
        public virtual string ShortName { get; set; }

        /// <summary>
        /// Описание
        /// </summary>
        public virtual string Description { get; set; }

        /// <summary>
        /// Код ОКЕИ
        /// </summary>
        public virtual string OkeiCode { get; set; }
    }
}
